packages/babel-helper-define-map/package.json

```diff
@@ -15,6 +15,6 @@
   "dependencies": {
     "@babel/helper-function-name": "^7.10.4",
     "@babel/types": "^7.10.4",
-    "lodash": "^4.17.13"
+    "lodash.has": "^4.5.2"
```

Comment:
`lodash.has` is similar to `lodash.get`, and `lodash.get` appears in [You-Dont-Need-Lodash-Underscore](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore).

That said, the [YDNLU native JavaScript replacement](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get) is relatively complex (9 lines of non-trivial code including regular expressions) and generates an error when key paths do not exist.  On balance it may make sense to maintain readability for now, although that's just my two cents and I'd welcome feedback.

Reply:
Let's keep it, since we are using it with a deep path.